packages/sdk/src/Message.ts

```diff
@@ -1,8 +1,24 @@
 import { HexString, StreamID } from '@streamr/utils'
-import { SignatureType, StreamMessage } from './protocol/StreamMessage'
+import { StreamMessage } from './protocol/StreamMessage'
+import { SignatureType } from '@streamr/trackerless-network'
+import { identityConfig, KeyType } from './identity/identityConfig'
+
+// Lookup structure for converting SignatureType to KeyType string
+export type MessageSignatureType = KeyType | 'ECDSA_SECP256K1_LEGACY' | 'ERC_1271'
```

Comment:
It is good to use string literal instead of TypeScript enum. For string literals the prefer lowercase format. The literals here weren't in that format before this PR, but maybe good to change now as this is anyway a breaking change.

```suggestion
export type MessageSignatureType = KeyType | 'ecdsaSecp256k1Legacy' | 'erc1271'
```

Same for literals in `KeyType`. 

Example of preferred usage of string literals in SDK: https://github.com/streamr-dev/network/blob/1025c2386fcd6f9a1cde970597770dc53efab84b/packages/sdk/src/Config.ts#L243-L244

Reply:
Deferring this to the discussion about which is better, unified naming or following convention